### PR TITLE
Add MSVC 16 & Clang 8

### PR DESCRIPTION
**Pull request by Croydon**
Closing https://github.com/conan-community/community/issues/217

### CODE REVIEW

**Comment by Croydon** [comment]
Creating for testing/ branch,
if successful please merge into stable/ as well

**Comment by Croydon** [comment]
Needs https://github.com/conan-community/conan-paho-c/pull/6 first

**Comment by SSE4** [comment]
weird error:
```
CMake Error at /usr/share/cmake-3.12/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Threads (missing: Threads_FOUND)
```

**Comment by uilianries** [comment]
It's the lack of support from that distro :disappointed:

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +33,7 @@ matrix:
       - <<: *linux
         env: CONAN_CLANG_VERSIONS=7.0 CONAN_DOCKER_IMAGE=conanio/clang7
       - <<: *linux
-        env: CONAN_GCC_VERSIONS=7 CONAN_DOCKER_IMAGE=conanio/gcc7-armv7hf CONAN_ARCHS=armv7hf
+        env: CONAN_CLANG_VERSIONS=8 CONAN_DOCKER_IMAGE=conanio/clang8
```
Comment (uilianries) [inline .travis.yml]:
```suggestion
        env: CONAN_CLANG_VERSIONS=8 CONAN_DOCKER_IMAGE=conanio/clang8 CONAN_DOCKER_32_IMAGES=1
```

**Comment by Croydon** [comment]
It's merge ready now

**Comment by Croydon** [comment]
please also merge in stable afterwards, thanks!

**Pull request merged**